lib/components/page.dart

```diff
@@ -55,8 +62,17 @@ class BccmPage extends ConsumerStatefulWidget {
 class _BccmPageState extends ConsumerState<BccmPage> {
   GlobalKey<State<FutureBuilder<Query$Page$page>>> futureBuilderKey = GlobalKey();
   Map<String, PaginationStatus<Fragment$ItemSectionItem>> paginationMap = {};
-  ScrollController scrollController = ScrollController();
   bool loadingBottomSectionItems = false;
+  late ScrollController scrollController;
+
+  @override
+  void initState() {
+    super.initState();
+    scrollController = widget.scrollController ?? ScrollController();
+    globalEventBus.on<ScrollToTopRequestEvent>().listen((event) {
+    TabsPage.values.byName(event.tab).name == event.tab ? scrollController.animateTo(0, duration: const Duration(milliseconds: 300), curve: Curves.decelerate) : null;
```

Comment:
but the probelm s when I use 
![screenshot](https://user-images.githubusercontent.com/69714702/211832106-6187ea03-b80f-461e-a8c2-9a05ca26217f.png)

the tab.Tab.home cannot not be recognized correctly, so I have to use the byName stuff

Furthermore, if I only use the tab.Tab.home, it will not be including the Search tab also, as I assume that the Serach should also have the same behavior

Reply:
at the home and search.dart file, I just put like 
    event.tab == 'home' ? scrollController?.animateTo(0, duration: const Duration(milliseconds: 300), curve: Curves.decelerate) : null;
is it ok?